<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcardMain</class>
 <widget class="QMainWindow" name="addcardMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Lable_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Add a </string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_type">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>TBD</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Input_title">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Title of your card</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Input_prob">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_prob">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>TBchange</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_ans">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>TBchange</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Input_ans">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Butconfirm">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AddÔºÅ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Butcancel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
